src/main/webapp/app/application/application-detail-for-applicant/application-detail-for-applicant.component.scss

```
@@ -1,25 +1,168 @@
 @use '../../../content/scss/breakpoints' as bp;
 
-$detail-card-2-column-max-width: bp.$xl;
+// Base card styles - shared between all card types
+.card {
+  border: 0.05rem solid var(--p-border-default);
+  border-radius: 0.5rem;
+  padding: 1.5rem;
+  margin-bottom: 2rem;
+}
+
+// Detail card extends base card with additional structure
+.detail-card {
+  @extend .card;
 
-.text-primary {
-  color: var(--p-text-primary);
+  .card-header {
+    display: flex;
+    align-items: center;
+    gap: 0.75rem;
+    border-bottom: 0.05rem solid var(--p-border-default);
+    margin-bottom: 1rem;
+    padding-bottom: 1rem;
```

I noticed the following issue when entering long texts in the editor component:
https://github.com/user-attachments/assets/94d5124d-c0c8-47a3-9a29-a9a40365ab49
The same issue is displayed in the summary page of the application creation form. A fix to this was previously adding "overflow-wrap: anywhere" to the text-card so it should be fixed by this addition
```suggestion
    padding-bottom: 1rem;
    overflow-wrap: anywhere;
```

---

Thank you for pointing that out. I missed that